README.md

```diff
@@ -2,8 +2,56 @@
 
 # Alloy
 
-Alloy is the code name for the Adobe Experience Platform Web SDK. It allows for streaming data into the platform, syncing identities, personalizing content, and more.
+Alloy is the code name for the Adobe Experience Platform Web SDK. It allows for recording events into the Adobe Experience Platform, syncing identities, personalizing content, and more.
 
 For documentation on how to use Alloy, please see the [user documentation](https://adobe.ly/36dGGp6).
 
 For documentation on how to contribute to Alloy, please see the [developer documentation](https://github.com/adobe/alloy/wiki).
+
+## Installation
+
+There are three supported ways to use Alloy.
+  1. Using Adobe Launch, install the Adobe Experience Platform Web SDK Extension.
+  2. Use the pre-built, minified version available via a CDN. You could also self-host this version.
+  3. Use the NPM library which exports an ES6 module.
```

Comment:
Yes, I think some of this should go in the user docs. I added more info here because the README.md is uploaded as part of the NPM package, and I wanted a quick guide to using it. Thinking about it more now, I think I should just include the ES6 instructions here, and then link to the user docs for the other way of using it.

Reply:
I think I'd still prefer having it all in the user docs, but maybe just split into different pages. Something like...

* Installing the SDK Using Launch
* Installing the SDK Using Standalone Library
* Installing the SDK Using NPM Package

or something like that. I like the idea of having all our customer-oriented docs in one place. I think it would be simpler for us to maintain and simpler for our customers to discover.